data-prepper-plugins/opensearch/README.md

```diff
@@ -155,6 +157,15 @@ If a single record turns out to be larger than the set bulk size, it will be sen
 
 - `trace_analytics_service_map`: No longer supported starting Data Prepper 2.0. Use `index_type` instead.
 
+### <a name="aws_configuration">AWS Configuration</a>
+
+The AWS configuration is the same for both SQS and S3.
```

Comment:
I do not think we should include the S3 portion. What if one changes over time? We might forget to update this documentation.

Reply:
I will remove that statement.